src/Admin/components/SearchBar/seachBarStyles.js

```diff
@@ -122,6 +127,14 @@ export default makeStyles((theme) => ({
     display: "inline",
 
   },
+  searchbarico: {
```

Comment:
I think we need not have this effect for search icon as it doesn’t have any action.